test/functional/wallet_listsinceblock.py

```diff
@@ -47,6 +47,7 @@ def test_no_blockhash(self):
             "category": "receive",
             "amount": 1,
             "blockhash": blockhash,
+            "blockheight": 102,
```

Comment:
In commit "rpc: Expose block height of wallet transactions" (19d2f1ae1091c94cc45bac3b258221785a454f90)

Would be better to also get this height from blockhash above instead of hardcoding it, if possible

Reply:
Ops, sorry for missing this one, fixed.